src/main/scala/org/constellation/domain/redownload/RedownloadPlan.scala

```diff
@@ -0,0 +1,14 @@
+package org.constellation.domain.redownload
+
+import cats.implicits._
+import org.constellation.domain.redownload.RedownloadService.SnapshotsAtHeight
+
+case class RedownloadPlan(
+  toDownload: SnapshotsAtHeight,
+  toRemove: SnapshotsAtHeight,
+  toLeave: SnapshotsAtHeight,
+) {
+  // TODO: Is it the proper place/way of calculating the expected state of acceptedSnapshots after redownload?
```

Comment:
@marcinwadon please take a look at all the TODO comments

Reply:
Sure. In relation to this one - it looks ok. As long as `toLeave` will have only those one, which are accepted and are aligned, it will be fine.